test/fixtures/projects/flatStructure.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const path = require('path');
+const android = require('./android');
+
+module.exports = {
+  android: android.valid,
+  node_modules: {
+    'react-native-vector-icons': {
+      'package.json': fs.readFileSync(path.join(__dirname, '../files/dependency.json')),
+      android: android.valid,
+    },
+    'rnpm-android-no-package': {
+      'package.json': fs.readFileSync(path.join(__dirname, '../files/dependency.json')),
+      android: android.noPackage,
+    },
```

Comment:
This was scaffolded super quickly and can be probably extracted directly to `test-cases`. To me it does not matter, but I can imagine the 2nd approach being more robust in a bigger case.